NEWS.md

```diff
@@ -9,6 +9,7 @@
 # Bug fixes
 
 * Fixed a bug when the filter panel overview would not refresh if the panel was hidden during a transition between active modules.
+* Fixed a bug where `varlabels` were assumed to be a character in `FilterStates` (now it is a character vector) and a `||` was used.
```

Comment:
In my opinion it is an enhancement not a bug fix as the method was working properly under the defined in documentation usage.

Reply:
moved to info part